scrapy/utils/iterators.py

```diff
@@ -125,7 +131,7 @@ def _getrow(csv_r):
 
 
 def _body_or_str(obj, unicode=True):
-    assert isinstance(obj, (Response, six.string_types)), \
+    assert isinstance(obj, (Response, six.string_types, six.binary_type)), \
```

Comment:
a nitpick: six.binary_type is only needed for Python 2.5 which doesn't have `bytes`